securedrop_client/logic.py

```diff
@@ -304,11 +304,16 @@ def completed_api_call(self, thread_id, user_callback):
 
     def login(self, username, password, totp):
         """
-        Given a username, password and time based one-time-passcode (TOTP),
-        create a new instance representing the SecureDrop api and authenticate.
+        Given a username, password and time based one-time-passcode (TOTP), create a new instance
+        representing the SecureDrop api and authenticate.
+
+        Default to 60 seconds until we implement a better request timeout strategy. We lower the
+        default_request_timeout for Queue API requests in ApiJobQueue in order to display errors
+        faster.
         """
         storage.mark_all_pending_drafts_as_failed(self.session)
-        self.api = sdclientapi.API(self.hostname, username, password, totp, self.proxy)
+        self.api = sdclientapi.API(
+            self.hostname, username, password, totp, self.proxy, default_request_timeout=60)
```

Comment:
Right now this is altering the default request timeouts for deletions but only temporarily until we implement the `DeletionJob` ✅

Reply:
Yep, this alters the timeout used during login (`self.api.authenticate`) deletion (`self.api.delete_source`), and logout (`self.api.logout`)